stem-cells-translational-medicine.csl

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" page-range-format="expanded" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Stem Cells Translational Medicine</title>
+    <title-short>Stem Cells Transl Med</title-short>
+    <id>http://www.zotero.org/styles/stem-cells-translational-medicine</id>
+    <link href="http://www.zotero.org/styles/stem-cells-translational-medicine" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-vancouver" rel="template"/>
+    <link href="http://www.stemcellsportal.com/sctm-info-for-contributors#_Toc372810938" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="biology"/>
+    <category field="medicine"/>
+    <issn>2157-6564</issn>
+    <eissn>2157-6580</eissn>
+    <summary>A style for Stem Cells Translational Medicine based on Elsevier Vancouver.</summary>
+    <updated>2017-01-14T17:42:41+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author">
+      <name delimiter-precedes-last="always" et-al-min="4" et-al-use-first="3" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
```

Comment:
@damnation333, does this et-al setting arrangement make the most sense? (there are also et-al setting on `<bibliography>`, but none in the "editor" macro)

It would make the style a little easier to read if there are only et-al settings on `<bibliography>`, or in the "author" and "editor" macros.